Files/UserControls/ModernSidebar.xaml.cs

```diff
@@ -118,11 +120,17 @@ private void NavigationViewItem_RightTapped(object sender, RightTappedRoutedEven
         {
             Microsoft.UI.Xaml.Controls.NavigationViewItem sidebarItem = (Microsoft.UI.Xaml.Controls.NavigationViewItem)sender;
             var item = sidebarItem.DataContext as LocationItem;
-            if (!item.IsDefaultLocation)
+            if (item.IsDefaultLocation)
             {
-                SideBarItemContextFlyout.ShowAt(sidebarItem, e.GetPosition(sidebarItem));
-                App.rightClickedItem = item;
+                UnpinItem.Visibility = Visibility.Collapsed;
```

Comment:
This shouldn't reference the xaml directly, it should instead set a value which is bound to the xaml.